### PR TITLE
Convert docs for operational model to ASCII LaTeX

### PR DESCRIPTION
**Pull request by lbianchi-lbl**
## Changes proposed in this PR:

- Convert Unicode characters to ASCII LaTeX for operational model only

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

File: docs/model_library/operational_water_management/index.rst [hunk 1]
```diff
@@ -359,68 +359,68 @@ The default objective function for this produced water operational model is to m
         \textcolor{red}{C^{TotalPiping}}+\textcolor{red}{C^{TotalStorage}}+\textcolor{red}{C^{TotalPadStorage}}+ \textcolor{red}{C^{TotalTrucking}}+\textcolor{red}{C^{Slack}-R^{TotalStorage}}
 
 
-**Completions Pad Demand Balance:** ∀p ∈ CP, t ∈ T
+**Completions Pad Demand Balance:** :math:`\forall p \in CP, t \in T`
```
Comment (tarnold17) [inline docs/model_library/operational_water_management/index.rst:362]:
```suggestion
**Completions Pad Demand Balance:** :math:`\forall \textcolor{blue}{p \in CP}, \textcolor{blue}{t \in T}`
```

If you want extra credit, you can add color to all of these statements like we have done in the strategic model documentation.

File: docs/model_library/operational_water_management/index.rst [hunk 2]
```diff
@@ -430,108 +430,108 @@ For each completions pad and time period, the volume of water (excluding freshwa
```
Comment (tarnold17) [inline docs/model_library/operational_water_management/index.rst:430]: For more extra credit, you can change line 427 to

    **Completions Pad Processing Capacity:** :math:`\forall \textcolor{blue}{p \in CP}, \textcolor{blue}{t \in T}`

File: docs/model_library/operational_water_management/index.rst [hunk 3]
```diff
@@ -651,185 +651,185 @@ The total stored water in a given time period must be less than the capacity. If
...
 .. math::
 
-    \textcolor{green}{ϵ^{Treatment}}⋅(\sum\nolimits_{(n,r)∈NRA}\textcolor{red}{F_{l,l,t}^{Piped}}+\sum\nolimits_{(s,r)∈SRA}\textcolor{red}{F_{l,l,t}^{Piped}}+\sum\nolimits_{(p,r)∈PRT}\textcolor{red}{F_{l,l,t}^{Trucked}}
+    \textcolor{green}{\varepsilon^{Treatment}} \cdot (\sum\nolimits_{(n,r)\in NRA}\textcolor{red}{F_{l,l,t}^{Piped}}+\sum\nolimits_{(s,r)\in SRA}\textcolor{red}{F_{l,l,t}^{Piped}}+\sum\nolimits_{(p,r)\in PRT}\textcolor{red}{F_{l,l,t}^{Trucked}}
 
-        +\sum\nolimits_{(p,r)∈CRT}\textcolor{red}{F_{l,l,t}^{Trucked}} )=\sum\nolimits_{(r,p)∈RCA}\textcolor{red}{F_{l,l,t}^{Piped}} + \textcolor{red}{F_{r,t}^{UnusedTreatedWater}}
+        +\sum\nolimits_{(p,r)\in CRT}\textcolor{red}{F_{l,l,t}^{Trucked}} )=\sum\nolimits_{(r,p)\in RCA}\textcolor{red}{F_{l,l,t}^{Piped}} + \textcolor{red}{F_{r,t}^{UnusedTreatedWater}}
 
-where :math:`\textcolor{green}{ϵ^{Treatment}}` <1
+where :math:`\textcolor{green}{\varepsilon^{Treatment}}` \le 1
```
Comment (tarnold17) [inline docs/model_library/operational_water_management/index.rst:764]:
```suggestion
where :math:`\textcolor{green}{\varepsilon^{Treatment}} \leq 1`
```

File: docs/model_library/operational_water_management/index.rst [hunk 4]
```diff
@@ -878,89 +878,89 @@ Assumptions:
...
+    \textcolor{green}{\varepsilon_{r,w}^{Treatment}} \cdot (\sum\nolimits_{(n,r)\in NRA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{n,w,t}} +\sum\nolimits_{(s,r)\in SRA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{s,w,t}}
 
-    +\sum\nolimits_{(p,r)∈PRT}\textcolor{purple}{F_{l,l,t}^{Trucked}}⋅\textcolor{red}{Q_{p,w,t}} +\sum\nolimits_{(p,r)∈CRT}\textcolor{purple}{F_{l,l,t}^{Trucked}}⋅\textcolor{red}{Q_{p,w,t}} )
+    +\sum\nolimits_{(p,r)\in PRT}\textcolor{purple}{F_{l,l,t}^{Trucked}} \cdot \textcolor{red}{Q_{p,w,t}} +\sum\nolimits_{(p,r)\in CRT}\textcolor{purple}{F_{l,l,t}^{Trucked}} \cdot \textcolor{red}{Q_{p,w,t}} )
 
-    = \textcolor{red}{Q_{r,w,t}}⋅(\sum\nolimits_{(r,p)∈RCA}\textcolor{purple}{F_{l,l,t}^{Piped}} + \textcolor{purple}{F_{r,t}^{UnusedTreatedWater}})
+    = \textcolor{red}{Q_{r,w,t}} \cdot (\sum\nolimits_{(r,p)\in RCA}\textcolor{purple}{F_{l,l,t}^{Piped}} + \textcolor{purple}{F_{r,t}^{UnusedTreatedWater}})
 
-where :math:`\textcolor{green}{ϵ_{r,w}^{Treatment}}` <1
+where :math:`\textcolor{green}{\varepsilon_{r,w}^{Treatment}}` \le 1
```
Comment (tarnold17) [inline docs/model_library/operational_water_management/index.rst:936]:
```suggestion
where :math:`\textcolor{green}{\varepsilon_{r,w}^{Treatment}} \leq 1`
```

File: docs/model_library/operational_water_management/index.rst [hunk 5]
```diff
@@ -878,89 +878,89 @@ Assumptions:
...
 .. math::
 
-    \sum\nolimits_{(n,o)∈NOA}\textcolor{purple}{F_{l,l,t}^{Piped}}⋅\textcolor{red}{Q_{n,w,t}} +\sum\nolimits_{(s,o)∈SOA}\textcolor{purple}{F_{l,l,t}^{Piped}}⋅\textcolor{red}{Q_{s,w,t}} +\sum\nolimits_{(p,o)∈POT}\textcolor{purple}{F_{l,l,t}^{Trucked}}⋅\textcolor{red}{Q_{p,w,t}}
+    \sum\nolimits_{(n,o)\in NOA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{n,w,t}} +\sum\nolimits_{(s,o)\in SOA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{s,w,t}} +\sum\nolimits_{(p,o)\in POT}\textcolor{purple}{F_{l,l,t}^{Trucked}} \cdot \textcolor{red}{Q_{p,w,t}}
 
-    = \textcolor{red}{Q_{o,w,t}}⋅\textcolor{purple}{F_{o,t}^{BeneficialReuseDestination}}
+    = \textcolor{red}{Q_{o,w,t}} \cdot \textcolor{purple}{F_{o,t}^{BeneficialReuseDestination}}
 
 
 .. _model_terminology:
```
Comment (tarnold17) [inline docs/model_library/operational_water_management/index.rst:966]: For more extra credit, change line 981 to

    :math:`[\textcolor{blue}{t}]` **or** :math:`[\textcolor{blue}{t \in T}]` **:** This notation indicates that timing of capacity expansion has not yet been implemented.

**Review by tarnold17 — CHANGES_REQUESTED** [review]
Some further changes are needed. I have also provided some optional suggestions.

**Comment by lbianchi-lbl** [comment]
@tarnold17 Thanks once again for all your review comments. I think I've addressed everything except for the "extra credit". If that's OK, feel free to re-review in case I missed something directly related to the ASCII-fication, and we can merge this PR in.

File: docs/model_library/operational_water_management/index.rst [hunk 6]
```diff
@@ -354,73 +354,73 @@ The default objective function for this produced water operational model is to m
 
 .. math::
 
-    min = \textcolor{red}{C^{TotalSourced}}+\textcolor{red}{C^{TotalDisposal}}+\textcolor{red}{C^{TotalTreatment}}+\textcolor{red}{C^{TotalCompletionsReuse}}+
+    min \ \textcolor{red}{C^{TotalSourced}}+\textcolor{red}{C^{TotalDisposal}}+\textcolor{red}{C^{TotalTreatment}}+\textcolor{red}{C^{TotalCompletionsReuse}}+
```
Comment (tarnold17) [inline docs/model_library/operational_water_management/index.rst:357]:
```suggestion
    \min \ \textcolor{red}{C^{TotalSourced}}+\textcolor{red}{C^{TotalDisposal}}+\textcolor{red}{C^{TotalTreatment}}+\textcolor{red}{C^{TotalCompletionsReuse}}+
```

Comment (lbianchi-lbl) [reply docs/model_library/operational_water_management/index.rst]: Good eye, thanks. Fixed

**Review by tarnold17 — DISMISSED** [review]

**Pull request merged**